README.md

```diff
@@ -39,12 +42,17 @@ Azure DevOps: Add an agent phase and then add task "Leapwork" to this agent phas
 7.	Enter JUnit report file name. The default name is "report.xml".  
 8.	Enter schedule titles you want to run, each one must be entered from a new line.  
 9.	Enter schedule ids, each one must be entered from a new line. Using id is preferable, because schedule title can be changed.  
-10.	Azure Devops: Add Test build step "Publish Test Results" to your build definition.  
-Azure DevOps: Add a task " Publish Test Results " to the previously added agent phase.  
+10.	Azure DevOps: Add Test build step "Publish Test Results" to your build definition.  
+	Azure DevOps: Add a task " Publish Test Results " to the previously added agent phase.  
 11.	Choose JUnit report format. Enter JUnit report file name. It MUST be the same you've entered in point 5. 
 12.	Run your schedule and get results. Enjoy! 
 13.	Check build results in a detailed report, which can be found in Build summary  
 14.	Download logs if needed.  
 
 # Troubleshooting
 - If you catch an error "No such run [runId]!" after schedule starting, increase time delay parameter.
+
+# Screenshots
```

Comment:
what is reason to have screenshots in ReadMe file?

Reply:
Screenshots may help the user to understand how to use plugin.

Reply:
it cannot help if it does not work
![image](https://user-images.githubusercontent.com/72276668/133236496-3907a9d1-bc1a-4787-a6fa-ac189eeba791.png)

Reply:
It will work, once the branch is merged with the main branch. As the url for screenshot is not correct because it isn't there yet.
https://github.com/leapwork/Azure-DevOps-plugin/blob/main/images/screen1.png

you can refer the following url for reference.
https://github.com/leapwork/Azure-DevOps-plugin/blob/featureBranch_AzureExtension/images/screen1.png

Reply:
ok, I checked the original repository and the format. screenshots must be "embedded" so DevOps Azure shows them in marketplace properly